components/LandingPage/Hero/Hero.module.scss

```diff
@@ -57,8 +57,8 @@
     // Prevent image showing in front of menu dropdown
     @include z-index(under);
     clip-path: url(#shardClipPath);
-    width: 375px;
-    height: 375px;
+    width: 335px;
+    height: 335px;
```

Comment:
Reducing the width of the hero here is a stopgap measure until the hero refactor ticket is completed and we use percentages.

This value was decreased so that the extra padding didn't cause the hero to overflow rest of the page and cause horizontal scrolling.